### DISCUSSION TITLE
Multiple dynamic themes

### DISCUSSION THREAD

**Discussion by alimertcakar** [chakra-ui/chakra-ui#4875] [Q&A]
I want to switch between multiple themes some are user created while app is running, not just dark or light. 

I've tried copying theme to redux, but both shallow copying and deep copying causes errors in chakra code.

Shallow copying: Error at:` breakpoints.base = (_breakpoints$base = breakpoints.base) != null ? _breakpoints$base : "0px";`
Deep copying: `TypeError: Cannot read properties of undefined (reading 'useSystemColorMode')`

**_What I am trying to do is a menu where a user can edit theme colors etc. according to their preference_**

**Comment by chr-ge** [comment]
Take a look at [this answer](https://github.com/chakra-ui/chakra-ui/discussions/3873#discussioncomment-706396).
